low-level/solution/lr2/server/health.py

```diff
@@ -606,10 +607,15 @@ def get_nw_ports_info(self):
             )
             self.set_health_data(nic_info, health_status, description=desc,
                                  specifics=[specifics])
-            network_cable_data.append(nic_info)
+            if interface == 'lo':
```

Comment:
add comment plz, on why we are doing this

Reply:
Done